src/Linter/ProcessLinterProcessBuilder.php

```diff
@@ -50,6 +49,8 @@ public function build($path)
             array_unshift($arguments, '--php');
         }
 
-        return ProcessBuilder::create($arguments)->setPrefix($this->executable)->getProcess();
+        $process = new Process('\''.$this->executable.'\' \''.implode('\' \'', $arguments).'\'');
```

Comment:
extra escaping and escaping and concatenation and again and again is so unreadable...